apps/obsidian/src/components/NodeTypeSettings.tsx

```diff
@@ -119,115 +260,189 @@ const NodeTypeSettings = () => {
-                className="flex-1"
-                disabled={
-                  !templateConfig.isEnabled || !templateConfig.folderPath
-                }
-              >
-                <option value="">
-                  {!templateConfig.isEnabled || !templateConfig.folderPath
-                    ? "Template folder not configured"
-                    : "No template"}
-                </option>
-                {templateFiles.map((templateFile) => (
-                  <option key={templateFile} value={templateFile}>
-                    {templateFile}
-                  </option>
-                ))}
-              </select>
-              <button
-                onClick={() => confirmDeleteNodeType(index)}
-                className="mod-warning p-2"
-              >
-                Delete
-              </button>
-            </div>
-            {formatErrors[index] && (
-              <div className="text-error mt-1 text-xs">
-                {formatErrors[index]}
-              </div>
-            )}
+        <div
+          key={nodeType.id}
+          className="node-type-item hover:bg-secondary-lt flex cursor-pointer items-center justify-between p-2"
+          onClick={() => startEditing(index)}
+        >
+          <span>{nodeType.name}</span>
+          <div className="flex gap-2">
+            <button
+              className="icon-button"
+              onClick={(e) => {
+                e.stopPropagation();
+                startEditing(index);
+              }}
+              aria-label="Edit node type"
+            >
+              <div className="icon" ref={(el) => el && setIcon(el, "pencil")} />
+            </button>
+            <button
+              className="icon-button mod-warning"
+              onClick={(e) => {
+                e.stopPropagation();
+                confirmDeleteNodeType(index);
+              }}
+              aria-label="Delete node type"
+            >
+              <div className="icon" ref={(el) => el && setIcon(el, "trash")} />
+            </button>
           </div>
         </div>
       ))}
-      <div className="setting-item">
-        <div className="flex gap-2">
-          <button onClick={handleAddNodeType} className="p-2">
-            Add Node Type
-          </button>
+    </div>
+  );
+
+  const renderEditForm = () => {
+    if (!editingNodeType) return null;
+
+    return (
+      <div>
+        <div className="mb-4 flex items-center justify-between">
+          <h3 className="dg-h3">Edit Node Type</h3>
           <button
-            onClick={handleSave}
-            className={`p-2 ${hasUnsavedChanges ? "mod-cta" : ""}`}
-            disabled={
-              !hasUnsavedChanges || Object.keys(formatErrors).length > 0
-            }
+            onClick={handleCancel}
+            className="icon-button mod-warning"
+            aria-label="Close edit form"
           >
-            Save Changes
+            <div className="icon" ref={(el) => el && setIcon(el, "x")} />
```

Comment:
Let's change this to something like `<-`.
`x` feels like I'm deleting something in this case, especially scary when editing a node.